test/res.sendFile.js

```diff
@@ -615,7 +615,7 @@ describe('res', function(){
         .get('/')
         .end(function(err, res){
           res.text.should.equal('<p>{{user.name}}</p>');
-          res.headers.should.have.property('content-type', 'text/html; charset=UTF-8');
+          should(res.headers).have.property('content-type', 'text/html; charset=UTF-8');
```

Comment:
Can this be changed to just use the features of `supertest` instead of `should`? We needed to do this a long time ago, so no better time to do this than now :)